avocado/utils/cloudinit.py

```diff
@@ -81,6 +91,9 @@ def iso(output_path, instance_id, username=None, password=None,
     :param phone_home_port: the port acting as an HTTP phone home
                             server that the instance should contact
                             once it has finished booting
+    :param authorized_key: a SSH public key to be added as an authorized key
+                           for the default user, similar to "ssh-rsa ..."
+    :type authorized_keys: str
```

Comment:
Only `authorized_key` has type defined here in the docs. Is it expected? If it is I think there's a typo on `type`: `s/authorized_keys/authorized_key/`

Reply:
We can always improve the docstrings, and add types for instance - I chose to add here because it may not be clear, as other such as "username", what should be given here.  Fixing the typo, though.